docs/interpreter/spark.md

```diff
@@ -140,6 +140,11 @@ You can also set other Spark properties which are not listed in the table. For a
     <td>true</td>
     <td>Import implicits, UDF collection, and sql if set true.</td>
   </tr>
+  <tr>
+    <td>zeppelin.spark.enableSupportedVersionCheck</td>
+    <td>true</td>
```

Comment:
I don't feel strongly about it, but since this is a debug config and it's turning off something important, I feel it is better to default to "off" that to disable it needs to be explicitly done?

I'll let other comment..